linkml/generators/shaclgen.py

```diff
@@ -198,15 +198,16 @@ def st_node_pv(p, v):
                         add_simple_data_type(prop_pv, r)
                     if s.pattern:
                         prop_pv(SH.pattern, Literal(s.pattern))
```

Comment:
Should these lines should also be moved in the same way? I'm not sure if only one `pattern` should be expected for a slot or if there can be multiple `patterns` in an `any_of`